vue-app/src/App.vue

```diff
@@ -80,8 +80,8 @@ export default class App extends Vue {
     setInterval(() => {
       this.$store.dispatch(LOAD_ROUND_INFO)
     }, 60 * 1000)
-    setInterval(() => {
-      this.$store.dispatch(LOAD_RECIPIENT_REGISTRY_INFO)
+    setInterval(async () => {
+      await this.$store.dispatch(LOAD_RECIPIENT_REGISTRY_INFO)
```

Comment:
Why should a dispatch be async?

Reply:
I just saw that LOAD_RECIPIENT_REGISTRY_INFO was async, so was using the await for that. I will remove.

Reply:
Ah, right, originally I was thinking no dispatches should be async but I was wrong about that... https://vuex.vuejs.org/guide/actions.html#composing-actions